gradle-module-plugin/src/functionalTest/kotlin/io/ia/sdk/gradle/modl/task/SignModuleTest.kt

```diff
@@ -188,25 +176,49 @@ class SignModuleTest : BaseTest() {
             "--keystoreFile=${signResources.keystore}",
             "--certFile=${signResources.certFile}",
             "--certAlias=selfsigned",
+            // --keystorePassword is not required by :signModule task for most
+            // types of keystores, though can add an integrity check on ks load
```

Comment:
Even PKCS#12 keystores don't care much about the keystore password on a simple `KeyStore.load`, only the cert password.